### PR TITLE
mod data to structured logging

### PR DESCRIPTION
**Pull request by chuntaochen**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
part of kubernetes/enhancements#1602

Which issue(s) this PR fixes:
Ref:

https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

Does this PR introduce a user-facing change?

```release-note 
Migrate pkg/ssh/ssh.go logs to structured logging

### CODE REVIEW

File: pkg/ssh/ssh.go [hunk 1]
```diff
@@ -333,8 +333,8 @@ func (l *SSHTunnelList) delayedHealthCheck(e sshTunnelEntry, delay time.Duration
 		defer runtime.HandleCrash()
 		time.Sleep(delay)
 		if err := l.healthCheck(e); err != nil {
-			klog.Errorf("Healthcheck failed for tunnel to %q: %v", e.Address, err)
-			klog.Infof("Attempting once to re-establish tunnel to %q", e.Address)
+			klog.ErrorS(err, "Healthcheck failed for tunnel to ", e.Address)
```
Comment (lala123912) [inline pkg/ssh/ssh.go:336]: Error formate,  the parameter ` e.Address` should be format of key-value. It's better to start before read  kubernetes/enhancements#1602.

File: pkg/ssh/ssh.go [hunk 2]
```diff
@@ -333,8 +333,8 @@ func (l *SSHTunnelList) delayedHealthCheck(e sshTunnelEntry, delay time.Duration
 		defer runtime.HandleCrash()
 		time.Sleep(delay)
 		if err := l.healthCheck(e); err != nil {
-			klog.Errorf("Healthcheck failed for tunnel to %q: %v", e.Address, err)
-			klog.Infof("Attempting once to re-establish tunnel to %q", e.Address)
+			klog.ErrorS(err, "Healthcheck failed for tunnel to", "address", e.Address)
```
Comment (thockin) [inline pkg/ssh/ssh.go:336]: Please drop "to" from the end of these

File: pkg/ssh/ssh.go [hunk 3]
```diff
@@ -371,17 +371,17 @@ func (l *SSHTunnelList) removeAndReAdd(e sshTunnelEntry) {
...
 }
 
 func (l *SSHTunnelList) Dial(ctx context.Context, net, addr string) (net.Conn, error) {
 	start := time.Now()
 	id := mathrand.Int63() // So you can match begins/ends in the log.
-	klog.Infof("[%x: %v] Dialing...", id, addr)
+	klog.InfoS("Dialing...", "id", id, "addr", addr)
 	defer func() {
-		klog.Infof("[%x: %v] Dialed in %v.", id, addr, time.Since(start))
+		klog.InfoS("Dialed in .", "id", id, "addr", addr, "timeDuration", time.Since(start))
```
Comment (thockin) [inline pkg/ssh/ssh.go:384]: remove the period

File: pkg/ssh/ssh.go [hunk 4]
```diff
@@ -443,11 +443,11 @@ func (l *SSHTunnelList) Update(addrs []string) {
 		for i := range l.entries {
 			if _, ok := wantAddrsMap[l.entries[i].Address]; !ok {
 				tunnelEntry := l.entries[i]
-				klog.Infof("Removing tunnel to deleted node at %q", tunnelEntry.Address)
+				klog.InfoS("Removing tunnel to deleted node at", "address", tunnelEntry.Address)
```
Comment (thockin) [inline pkg/ssh/ssh.go:446]: remove the "at"

File: pkg/ssh/ssh.go [hunk 5]
```diff
@@ -443,11 +443,11 @@ func (l *SSHTunnelList) Update(addrs []string) {
...
 				go func() {
 					defer runtime.HandleCrash()
 					if err := tunnelEntry.Tunnel.Close(); err != nil {
-						klog.Errorf("Failed to close tunnel to %q: %v", tunnelEntry.Address, err)
+						klog.ErrorS(err, "Failed to close tunnel to", "address", tunnelEntry.Address)
```
Comment (thockin) [inline pkg/ssh/ssh.go:450]: remove the "to"

File: pkg/ssh/ssh.go [hunk 6]
```diff
@@ -459,14 +459,14 @@ func (l *SSHTunnelList) Update(addrs []string) {
 }
 
 func (l *SSHTunnelList) createAndAddTunnel(addr string) {
-	klog.Infof("Trying to add tunnel to %q", addr)
+	klog.InfoS("Trying to add tunnel to", "addr", addr)
```
Comment (thockin) [inline pkg/ssh/ssh.go:462]: remove "to"

Comment (thockin) [reply pkg/ssh/ssh.go:462]: same in all cases - I won't repeat furtther

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Comment by GreenApple10** [comment]
/release-note mod pkg/ssh/ssh.go logs to structured logging.

**Comment by ehashman** [comment]
/triage accepted

someone will need to fix the release note for this to merge

**Comment by GreenApple10** [comment]
/release-note mod pkg/ssh/ssh.go logs to structured logging.

**Comment by ehashman** [comment]
/hold

This needs a release note.

**Comment by GreenApple10** [comment]
/release-note mod pkg/ssh/ssh.go logs to structured logging.

**Comment by GreenApple10** [comment]
> /hold
> 
> This needs a release note.

How can I add the release note?  I've written it in comment area and try use label to add it, but i failed.    I  am dying for your help.

**Comment by ehashman** [comment]
Hi @GreenApple10,

Your note block must begin with ```release-note in your first comment. Like this:

```release-note
Migrate pkg/ssh/ssh.go logs to structured logging
```

You can quote my reply to see the markdown.

**Comment by GreenApple10** [comment]
> Hi @GreenApple10,
> 
> Your note block must begin with ```release-note in your first comment. Like this:
> 
> ```
> Migrate pkg/ssh/ssh.go logs to structured logging
> ```
> 
> You can quote my reply to see the markdown.

Thanks !

**Pull request closed without merging**